### ISSUE TITLE
Feature: Sync Simkl completed status

### ISSUE THREAD

**Issue by abarnwell** [SIMKL/Emby#10]
Would it be possible to monitor the JSON feed for your completed movies and TV shows provided by SIMKL so that if something appears on that list, it get's tagged as played in my Emby library? Would make it easier to keep the two in sync with one another. I think the Trackt plugin for Plex does something similar but realise the services are completely different.

**Comment by ekleop** [comment]
First we need to understand if there is such method in Emby's api for this. Some push or poll method then we can try to implement it.

**Comment by neomanyouth** [comment]
Just found this when I was searching for Simkl integration with Emby. Looks like the two API calls you'd need would BE:
http://swagger.emby.media/?staticview=true#/ItemUpdateService
http://swagger.emby.media/?staticview=true#/PlaystateService

**Comment by trivarialthea** [comment]
Is this still maintained?

**Comment by masyk** [comment]
> Is this still maintained?

Yes, there's a new update waiting to be worked on for multiuser support https://github.com/SIMKL/Emby/pull/18

**Comment by Bortus-AI** [comment]
Any update on this?

**Comment by ekleop** [comment]
Done in 2.0.0, released today. The plugin can import your Simkl history into Emby, everything completed or watched on Simkl is marked as played in your library, with the original watch date. Run it from your Simkl plugin page in Emby, or switch on automatic import and
pick how often Emby checks Simkl. 
Requires Emby 4.8.11 or newer.
